### DISCUSSION TITLE
reset scores in co-op games

### DISCUSSION THREAD

**Discussion by shayanshokry** [efroemling/ballistica#479] [Q&A] [answered]
how to reset "Your Best Scores" list in co-op game?
i have so many fake scores using modpacks and mods and want to delete them.
is there any way to do that?

**Comment by IvanPragma** [comment, marked as answer]
Switch to another account or maybe deleting BombSquad `config.json` will help

**Comment by MatiasMGz** [comment]
Please close this...
